pymc3/distributions/distribution.py

```diff
@@ -394,6 +396,13 @@ def _distr_parameters_for_repr(self):
         return []
 
 
+@_logprob.register(NoDistribution)
+def logp(op, value_var, *dist_params, **kwargs):
+    _dist_params = dist_params[3:]
+    value_var = value_var[0]
```

Comment:
Same thing with the `value_var` variable name. (mypy would also complain about it, because overwriting the variable name changes the type...)